monitor/events.go

```diff
@@ -234,8 +260,19 @@ func LogSegmentTranscodeFailed(subType string, nonce, seqNo uint64, err error) {
 	detectSeqDif(props, nonce, seqNo)
 
 	sendPost("SegmentTranscodeFailed", nonce, props)
+
+	if subType == SegmentTranscodeErrorNoOrchestrators {
+		// this will allow us to find out at what stages of stream "NoOrchestrators" errors occur
+		props2 := map[string]interface{}{
+			"subType": fmt.Sprintf("NoOrchestrators-seq-%04d", seqNo),
```

Comment:
I'm still not totally sold on double-counting the errors in this way. Can you address the concerns that were raised here? https://github.com/livepeer/go-livepeer/pull/797#discussion_r267575117